src/pages/Contributing/GitConventions/WIPStatus/index.md

```diff
@@ -0,0 +1,8 @@
+---
+title: 'WIP Status'
+order: 4
+---
+
+I don't know what we should say here just that we allow to put a WIP status on PR.
```

Comment:
add [WIP] in front of your PR name and using a label
Wip will not be reviewed unless you specifically ask someone to